packages/origin-backend-core/src/OrganizationInvitation.ts

```diff
@@ -20,11 +20,9 @@ export interface IOrganizationInvitationProperties {
 }
 
 export interface IOrganizationInvitation extends IOrganizationInvitationProperties {
-    organization: IOrganization | IOrganization['id'];
-}
-
-export interface IOrganizationInvitationWithRelations extends IOrganizationInvitation {
     organization: IOrganization;
+    sender: string;
+    processed: boolean;
```

Comment:
Can we use the status of the invitation to represent if invitation was received or not?

So IMHO adding extra statuses to already existing enum should solve the problem:

OrganizationInvitationStatus.Accepted
OrganizationInvitationStatus.Rejected
OrganizationInvitationStatus.Sent
OrganizationInvitationStatus.Delivered

Is there a use case which requires us to have more than 1 status at time?